crates/client/sync/src/l2.rs

```diff
@@ -52,15 +53,15 @@ pub struct L2VerifyApplyConfig {
     flush_every_n_blocks: u64,
     flush_every_n_seconds: u64,
     stop_on_sync: bool,
-    telemetry: TelemetryHandle,
+    telemetry: Arc<TelemetryHandle>,
```

Comment:
telemetryhandle is already an arc

Reply:
Not anymore, this had to be updated to make it so the service can be restarted.